### PR TITLE
Added Table and TableRow component

### PR DESCRIPTION
**Pull request by rishabh-ti**

### CODE REVIEW

File: src/components/editor/ComponentPreview.tsx [hunk 1]
```diff
@@ -215,6 +227,11 @@ const ComponentPreview: React.FC<{
       return <StatHelpTextPreview component={component} />
     case 'StatGroup':
       return <StatGroupPreview component={component} />
+    case 'Table':
+    case 'TableContainer':
```
Comment (vinayak-sachdeva) [inline src/components/editor/ComponentPreview.tsx]: table container is mentioned above as well

File: src/componentsList.ts [hunk 2]
```diff
@@ -113,6 +113,22 @@ export const menuItems: MenuItems = {
...
+      TableCaption: {},
+      Thead: {},
+      Tbody: {},
+      Tfoot: {},
+      Tr: {},
+      Th: {},
+      Td: {},
+    },
+  },
+  TableRow: {
```
Comment (vinayak-sachdeva) [inline src/componentsList.ts:129]: what component is this?

**Review by vinayak-sachdeva — CHANGES_REQUESTED** [review]

File: src/componentsList.ts (hunk 2 above)
Comment (rishabh-ti) [reply src/componentsList.ts:129]: To add another row to the table, added it as a separate Meta component.

File: src/components/editor/ComponentPreview.tsx (hunk 1 above)
Comment (rishabh-ti) [reply src/components/editor/ComponentPreview.tsx]: Yes coz Table is included in a Table container, so when we are dragging the Meta component, we want the table inside the TableContainer. Check TablePreview for that.

Comment (vinayak-sachdeva) [reply src/components/editor/ComponentPreview.tsx]: The code is unreachable for TableContainer. Check line 153

File: src/componentsList.ts (hunk 2 above)
Comment (vinayak-sachdeva) [reply src/componentsList.ts:129]: Couldn't find it in MetaComponents. Only TableMeta and TableRowMeta are there

Comment (rishabh-ti) [reply src/componentsList.ts:129]: Its there - TableRowMeta maps to TableRow! Check builder.ts file to see how it is getting built

File: src/components/editor/ComponentPreview.tsx (hunk 1 above)
Comment (rishabh-ti) [reply src/components/editor/ComponentPreview.tsx]: Fixed in latest commit

**Review by vinayak-sachdeva — APPROVED** [review]

**Pull request merged**